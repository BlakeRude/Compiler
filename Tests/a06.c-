// C-S21
int x;

main()
{
     x = 273;
     output(x);
}

