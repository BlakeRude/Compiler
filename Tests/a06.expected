Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: a06.tm
273
Bye.
